### ISSUE TITLE
Empty exclude-folders appears to filter out all output

### ISSUE THREAD

**Issue by allenporter** [k8s-at-home/renovate-helm-releases#14]
Renovate helm releases appears to have stopped for my repository (tracked in my [issue](https://github.com/allenporter/k8s-gitops/issues/190) though I don't have a specific date.  It seems to be related to `excluded-folders`

I can run renovate locally and it works:
```
$ ./renovate.py --cluster-path tests/ --debug
2021-06-08 07:59:47 Renovate Helm Releases INFO     Found HelmRepository 'flux-system/traefik-charts' url 'https://helm.traefik.io/traefik'
...
```

My github action is configured like this:
```
      - name: Renovate Helm Releases
        uses: k8s-at-home/renovate-helm-releases@v1
        with:
          cluster-path: './'
          debug: 'yes'
```
And for an [example run](https://github.com/allenporter/k8s-gitops/runs/2774683904?check_suite_focus=true) runs the docker image with these flags:

```
/usr/bin/docker run ...  ghcr.io/k8s-at-home/renovate-helm-releases:v1.4.1  "./" "" "yes" "no" "no"
```

I modified `entrypoint.sh` to work in my environment, so then running it with the above flags produced no output:

```
$ bash -x ./entrypoint.sh "./tests/" "" "yes" "no" "no"
+ CLUSTER_PATH=./tests/
+ EXCLUDE_FOLDERS=
+ DEBUG=yes
+ DRY_RUN=no
+ TOLERATE_YAML_ERRORS=no
+ '[' yes = yes ']'
+ DEBUG=--debug
+ '[' no = yes ']'
+ DRY_RUN=
+ '[' no = yes ']'
+ TOLERATE_YAML_ERRORS=
+ python3 ./renovate.py --cluster-path=./tests/ --excluded-folders= --debug

```

However, running this with bogus excluded folders does make it start working:
```
$ bash -x ./entrypoint.sh "./tests/" "some-excluded-folders" "yes" "no" "no"
+ CLUSTER_PATH=./tests/
+ EXCLUDE_FOLDERS=some-excluded-folders
+ DEBUG=yes
+ DRY_RUN=no
+ TOLERATE_YAML_ERRORS=no
+ '[' yes = yes ']'
+ DEBUG=--debug
+ '[' no = yes ']'
+ DRY_RUN=
+ '[' no = yes ']'
+ TOLERATE_YAML_ERRORS=
+ python3 ./renovate.py --cluster-path=./tests/ --excluded-folders=some-excluded-folders --debug
2021-06-08 08:04:12 Renovate Helm Releases INFO     Found HelmRepository 'flux-system/traefik-charts' url 'https://helm.traefik.io/traefik'
2021-06-08 08:04:12 Renovate Helm Releases INFO     Found HelmRepository 'flux-system/k8s-at-home-charts' url 'https://k8s-at-home.com/charts/'
2021-06-08 08:04:12 Renovate Helm Releases INFO     Found HelmRepository 'flux-system/ingress-nginx-charts' url 'https://kubernetes.github.io/ingress-nginx'
2021-06-08 08:04:12 Renovate Helm Releases INFO     Found HelmRelease 'example/ingress-nginx-3' with repo 'flux-system/ingress-nginx-charts'
2021-06-08 08:04:12 Renovate Helm Releases DEBUG    Skipping 'example/ingress-nginx-3': No 'sourceRef' in spec.chart.spec
2021-06-08 08:04:12 Renovate Helm Releases INFO     Found HelmRelease 'default/blocky' with repo 'flux-system/k8s-at-home-charts'
2021-06-08 08:04:12 Renovate Helm Releases INFO     Found HelmRelease 'default/ingress-nginx-2' with repo 'flux-system/ingress-nginx-charts'
2021-06-08 08:04:12 Renovate Helm Releases WARNING  Skipping 'cert-manager/cert-manager': No HelmRepository for 'flux-system/jetstack-charts'
2021-06-08 08:04:12 Renovate Helm Releases INFO     Found HelmRelease 'default/ingress-nginx-1' with repo 'flux-system/ingress-nginx-charts'
2021-06-08 08:04:12 Renovate Helm Releases DEBUG    Skipping 'example/ingress-nginx-3': No 'chart' or 'version' in spec.chart.spec
2021-06-08 08:04:12 Renovate Helm Releases INFO     Updating 'flux-system/ingress-nginx-charts' renovate annotations in 'tests/helm-releases/ingress-nginx-3/overlay-values.yaml' with 'https://kubernetes.github.io/ingress-nginx'
2021-06-08 08:04:12 Renovate Helm Releases INFO     Updating 'flux-system/k8s-at-home-charts' renovate annotations in 'tests/helm-releases/blocky/helm-release.yaml' with 'https://k8s-at-home.com/charts/'
2021-06-08 08:04:12 Renovate Helm Releases INFO     Updating 'flux-system/ingress-nginx-charts' renovate annotations in 'tests/helm-releases/ingress-nginx-2/helm-release.yaml' with 'https://kubernetes.github.io/ingress-nginx'
2021-06-08 08:04:12 Renovate Helm Releases DEBUG    Skipping 'cert-manager/cert-manager': Could not determine repo
2021-06-08 08:04:12 Renovate Helm Releases INFO     Updating 'flux-system/ingress-nginx-charts' renovate annotations in 'tests/helm-releases/ingress-nginx-1/helm-release.yaml' with 'https://kubernetes.github.io/ingress-nginx'
```

**Comment by allenporter** [comment]
My guess is that `--excluded-folders=` means there is a single empty string excluded folder which excludes everything.

```
$ python3
>>> from pathlib import Path
>>> excluded_folders = [""]
>>> excluded_files = [p for folder in excluded_folders for p in Path(folder).rglob("*")]
>>> len(excluded_files)
2014
```

So perhaps this should be handled in `entrypoint.sh` by omitting the flag

**Comment by bjw-s** [comment]
This should be fixed now with v1.4.2 of the Github action. Thanks for reporting and providing the fix!

**Comment by allenporter** [comment]
Confirmed working on my repo, thanks!
